create_xlinkdocs.sh

```diff
@@ -2,111 +2,81 @@
 # Build cross linked doxygen documents and load into buildbot public_html website
 
 SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
-source ${SCRIPT_DIR}/settings.cfg.sh
-source ${LSSTSW}/bin/setup.sh
+source "${SCRIPT_DIR}/settings.cfg.sh"
+source "${LSSTSW}/bin/setup.sh"
 
 usage() {
-    echo "Usage: $0 --type <type> --user <remote user on doxy host> --host <remote doxy host> --path  <remote doxy docs path>"
+    echo "Usage: $0 --type <type> --host <remote doxy host> --path <doxy docs path>"
```

Comment:
The intent was to remove it.  This script is really in a rough state (even before I touched it).  I'm working on cleaning it up a bit this morning but I want to get it shipped in the next hour or so.